server_installation/topics/operating-mode/crossdc.adoc

```diff
@@ -0,0 +1,736 @@
+
+**Automatically** - After some amount of failed backups, the `site2` will usually be taken offline automatically. This is done due the configuration of `take-offline` element inside the cache configuration as configured in link:jdgsetup[JDG server setup].
+
+```
+<take-offline min-wait="60000" after-failures="3" />
+```
+
+This example shows that the site will be taken offline automatically for the particular single cache if there are at least 3 subsequent failed backups and there is no any successful backup within 60 seconds.
+
+Automatically taking a site offline is useful especially if the broken network between sites is unplanned. The disadvantage is that there will be some failed backups until the network outage is detected, which could also mean  failures on the application side. 
+For example, there will be failed logins for some users or big login timeouts. Especially if `failure-policy` with value `FAIL` is used.
+
+WARNING: The tracking of whether a site is offline is tracked separately for every cache.
+ 
+
+.Take site online
+
+Once your network is back and `site1` and `site2` can talk to each other, you may need to put the site online. This needs to be done manually through JMX or CLI in similar way as taking a site offline.
+Again, you may need to check all the caches and bring them online.
+
+Once the sites are put online, it's usually good to:
+* Do the link:statetransfer[state transfer].
+* Manually link:clearcache[clear the Keycloak caches]. 
+
+
+[[statetransfer]]
+==== State transfer
+
+State transfer is a manually required step. JDG does not do this automatically, for example during split-brain, it is only the admin who may decide which site has preference and hence if state transfer needs to be done bidirectionally between both sites or just unidirectionally, as in only from `site1` to `site2`, but not from `site2` to `site1`. 
+
+A bidirectional state transfer will ensure that entities which were created *after* split-brain on `site1` will be transferred to `site2`. This is not an issue as they do not yet exist on `site2`. Similarly, entities created *after* split-brain on `site2` will be transferred to `site1`. Possibly problematic parts are those entities which exist *before* split-brain on both sites and which were updated during split-brain on both sites. When this happens, one of the sites will *win* and will overwrite the updates done during split-brain by the second site.
+
+Unfortunately, there is no any universal solution to this. Split-brains and network outages are just state, which is usually impossible to be handled 100% correctly with 100% consistent data between sites. In the case of {project_name}, it typically is not a critical issue. In the worst case, users will need to re-login again to their clients, or have the improper count of loginFailures tracked for brute force protection. See the JDG/JGroups/Infinispan docmentation for more tips how to deal with split-brain.
```

Comment:
docmentation -> documentation

Reply:
Hahahahaha! How embarrassing for the tech writer to misspell "documentation". LOL